app/src/components/ExportPane.js

```diff
@@ -32,61 +36,163 @@ const getFormattedResult = (category, filteredResults) => {
     : 'N/A';
 };
 
-function CategorySummary({ results, category }) {
-  const filteredResults = useMemo(() => results.filter((r) => r.Category === category), [results, category]);
+const SUMMARY_CATEGORIES = ['MEDICATION', 'ANATOMY', 'MEDICAL_CONDITION', 'TEST_TREATMENT_PROCEDURE'];
+const SUMMARY_CATEGORIES_SET = new Set(SUMMARY_CATEGORIES);
 
-  return <p>{getFormattedResult(category, filteredResults)}</p>;
-}
+const getFormattedCategorySummaries = (results) => {
+  const resultsByCategory = {};
+
+  SUMMARY_CATEGORIES.forEach((category) => {
+    resultsByCategory[category] = [];
+  });
+
+  results.forEach((result) => {
+    const category = result.Category;
+
+    if (SUMMARY_CATEGORIES_SET.has(category)) resultsByCategory[category].push(result);
+  });
+
+  return SUMMARY_CATEGORIES.map((category) => ({
+    category,
+    summary: getFormattedCategorySummary(category, resultsByCategory[category]),
+  }));
+};
+
+const getFilteredResults = (resultChunks, excludedItems) =>
+  [].concat(...resultChunks).filter((x) => !excludedItems.includes(x.id));
+
+const getExportSections = ({ resultChunks, excludedItems, soapSummary, transcriptChunks }) => {
+  const filteredResults = getFilteredResults(resultChunks, excludedItems);
+
+  const formattedCategorySummaries = getFormattedCategorySummaries(filteredResults);
+
+  return [
+    {
+      header: {
+        type: 'SECTION',
+        content: 'Summary',
+      },
+      content: [
+        `Thank you for visiting the clinic today, ${new Date()
+          .toISOString()
+          .slice(
+            0,
+            10,
+          )}. Please take a moment to review the following important information from today's consultation and reach out to us at +12345678910 if you have any questions.`,
+        soapSummary,
+      ],
+    },
+    ...formattedCategorySummaries.map(({ category, summary }) => ({
+      header: {
+        type: 'SUB_SECTION',
+        content: displayNames[category],
+      },
+      content: [summary],
+    })),
+    {
+      header: {
+        type: 'SECTION',
+        content: 'Visit Transcription',
+      },
+      content: [
+        'Below is the transcription for your visit',
+        ...(transcriptChunks ? transcriptChunks : []).map((chunk) => chunk.text),
+      ],
+    },
+  ];
+};
+
+const translateSection = (section, lang, jwtToken) => {
```

Comment:
While highly unlikely that the input may be too long ,  you should check if the input needs to be paginated.
The API only supports translation of up to 5k bytes in a call. See [here](https://github.com/aws-samples/medical-transcription-analysis/pull/53)